browser/brave_content_browser_client_browsertest.cc

```diff
@@ -166,7 +166,7 @@ IN_PROC_BROWSER_TEST_F(BraveContentBrowserClientTest, CanLoadChromeURL) {
 
 IN_PROC_BROWSER_TEST_F(BraveContentBrowserClientTest, CanLoadCustomBravePages) {
   std::vector<std::string> pages{
-      "adblock",
+      "ipfs-internals",
```

Comment:
After discussing with @petemill - We decided to replace adblock with ipfs-internals because Adblock behaves differently now and this test's purpose to check if custom brave pages are loaded.